spec/controllers/v1.0/workflows_controller_spec.rb

```diff
@@ -482,35 +482,34 @@
     let(:obj) { { :object_type => 'inventory', :app_name => 'topology', :object_id => '123'} }
 
     it 'returns status code 204' do
-      post "#{api_version}/workflows/#{id}/unlink", :params => obj, :headers => default_headers
+      post "#{api_version}/workflows/#{id}/unlink", :params => obj, :headers => default_headers, :as => :json
 
       expect(response).to have_http_status(204)
     end
   end
 
   # TODO: resolve needs further work to query tag names
-  describe 'POST /workflows/resolve' do
-    let(:obj_a) { { :object_type => 'inventory', :app_name => 'topology', :object_id => '123'} }
-    let(:obj_b) { { :object_type => 'portfolio', :app_name => 'catalog', :object_id => '123'} }
-
+  xdescribe 'POST /workflows/resolve' do
+    let(:obj_a) { { :object_type => 'ServiceInventory', :app_name => 'topology', :object_id => '123'} }
+    let(:obj_b) { { :object_type => 'Portfolio', :app_name => 'catalog', :object_id => '123'} }
     before do
       allow(AddRemoteTags).to receive(:new).with(obj_a).and_return(add_tag_svc)
       allow(add_tag_svc).to receive(:process).with(tag).and_return(add_tag_svc)
       allow(GetRemoteTags).to receive(:new).with(obj_a).and_return(get_tag_svc)
       allow(GetRemoteTags).to receive(:new).with(obj_b).and_return(get_tag_svc)
       allow(get_tag_svc).to receive(:process).and_return(get_tag_svc)
-      post "#{api_version}/workflows/#{id}/link", :params => obj_a, :headers => default_headers
+      post "#{api_version}/workflows/#{id}/link", :params => obj_a, :headers => default_headers, :as => :json
```

Comment:
@hsong-rh :as => :json should not be needed if the spec has the proper type the common gem should do this for you